### ISSUE TITLE
Not playing in background on FirefoxOS 2.2 (Flame)

### ISSUE THREAD

**Issue by gbraad** [nazar-pc/CleverStyle-Music#19]
# Steps to reproduce
* Device is in idle state
* Music is available in device storage and has been indexed into the library
* Open CleverStyle music
* Play any music from the library
* Press home and navigate away from the application

# Expected result
Music is playing in the background

# Actual result
Music stops

# Environment
* Firefox OS 2.2
* Flame
* mp3 media files
* latest version as of yet, published on the Firefox market place

# Notes
- Background play does work on Firefox for Android using CleverStyle.
- Background play is available on FirefoxOS as several games kepy playing annoying tunes in the background when navigating away, such as "block the pig".

**Comment by nazar-pc** [comment]
Thanks for reporting this, I'll try to fix that soon.

**Comment by nazar-pc** [comment]
I found some time to work on this app again. Can you check once again, and if you can still reproduce it - would you be so kind to point me to some game that works fine in background?

**Comment by gbraad** [comment]
Was the application update pushed to the Mozilla webstore?
Note: IIRC the game was called "Block the pig"

On Thu, Jul 16, 2015 at 9:48 PM, Nazar Mokrynskyi <notifications@github.com>
wrote:

> I found some time to work on this app again. Can you check once again, and
> if you can still reproduce it - would you be so kind to point me to some
> game that works fine in background?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nazar-pc/CleverStyle-Music/issues/19#issuecomment-121962155>
> .
>



-- 
Gerard Braad — 吉拉德
   F/OSS & IT Consultant in Beijing
   http://gbraad.nl  gpg: 0x592CFE75

**Comment by nazar-pc** [comment]
Yes, but that version is a bit buggy, there is an issue with switching to next track, updated version is not reviewed yet, but you can easily install app directly from this repository.
Thanks, I'll try to look at that game.

**Comment by nazar-pc** [comment]
Manifest of that game doesn't state any permission to play music in background, I have no idea how that may happen.
